pyresample/geometry.py

```diff
@@ -1074,11 +1131,16 @@ def _compute_bound_centers_numpy(self, proj_dict, lons, lats):
         y_is_pole = (ymax >= 90 - epsilon) or (ymin <= -90 + epsilon)
         if proj4.crs.is_geographic and x_passes_antimeridian and not y_is_pole:
             # cross anti-meridian of projection
-            xmin = np.nanmin(xarr[xarr >= 0])
-            xmax = np.nanmax(xarr[xarr < 0]) + 360
-        return xmin, ymin, xmax, ymax
+            if antimeridian_mode == "global_extents":
+                xmin, xmax = (None, None)
+            else:
+                xmin = np.nanmin(xarr[xarr >= 0])
+                xmax = np.nanmax(xarr[xarr < 0]) + 360
+            if antimeridian_mode == "modify_projection":
```

Comment:
Would `shift_prime_median` be more specific?

Reply:
Hm interesting idea. I think I chose "projection" in case other projections in the future may need different types of modifications. For example, if we were trying to support someone using `+x` and `+y` parameters in their projection definition, I'm not sure the `+pm` would be the best thing to modify.